umm-spec-lib/resources/json-schemas/variable/umm/v1.9/umm-var-search-results-json-schema.json

```diff
@@ -0,0 +1,286 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "ItemType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Represents a single item found in search results. It contains some metadata about the item found along with the UMM representing the item. umm won't be present if the item represents a tombstone.",
+      "properties": {
+        "meta": {
+          "$ref": "#/definitions/MetaType"
+        },
+        "umm": {
+          "$ref": "umm-var-json-schema.json"
+        },
+        "associations": {
+          "$ref": "#/definitions/AssociationsInItemType"
+        }
+      },
+      "required": ["meta"]
+    },
+    "MetaType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "CMR level metadata about the item found. This represents data not found in the actual metadata.",
+      "properties": {
+        "provider-id": {
+          "description": "The identity of the provider in the CMR",
+          "type": "string",
+          "minLength": 1,
+          "pattern": "[A-Z0-9_]+"
```

Comment:
add the provider id length limit

Reply:
I didn't change this file. I just brought it forward. Looking at the code base, this was in the initial release 6 years ago. This file is only used for running tests. I think I will leave it as is, as when we have to change the length of a provider then this is another thing that will needed to be changed.